tests/test_pause_resume.py

```diff
@@ -52,7 +52,7 @@ def setUp(self) -> None:
     # Can't use override_settings due to the copying of the settings values into
     # module values at startup.
     @mock.patch(
-        'django_lightweight_queue.app_settings.BACKEND',
+        'django_lightweight_queue.app_settings.Defaults.BACKEND',
         new='django_lightweight_queue.backends.redis.RedisBackend',
     )
```

Comment:
Hrm, I had expected that with the layers approach we'd be able to use `override_settings` (if we also moved to using Django's [`SimpleTestCase`](https://docs.djangoproject.com/en/3.2/topics/testing/tools/#simpletestcase) in place of `unittest.TestCase`). Does that not work?

Reply:
I haven't approached refactoring the tests since we still haven't nailed down the proper override process in a way that works for everyone yet. After that, it's probably better set aside for a future ticket later on down the line as long as the test does what it's supposed to do now.